package/src/components/Channel/__tests__/Channel.test.js

```diff
@@ -14,8 +14,10 @@ import {
   MessagesProvider,
 } from '../../../contexts/messagesContext/MessagesContext';
 import { ThreadContext, ThreadProvider } from '../../../contexts/threadContext/ThreadContext';
+
 import { useMockedApis } from '../../../mock-builders/api/useMockedApis';
 import { getOrCreateChannelApi } from '../../../mock-builders/api/getOrCreateChannel';
+import dispatchChannelDeletedEvent from '../../../mock-builders/event/channelDeleted';
```

Comment:
Interesting that this is a default export when all others aren't 🤔 Not important though, just looked a bit odd in the list

Reply:
Yeah kinda stupid, one of my ignorance in early stages :/

Reply:
I will try to replace them with non-default exports.